internal/dflog/loginit.go

```diff
@@ -90,12 +92,12 @@ func InitScheduler(verbose, console bool, dir string, rotateConfig LogRotateConf
 		},
 	}
 
-	return createFileLogger(verbose, meta, logDir, rotateConfig)
+	return createFileLogger(verbose, logLevel, meta, logDir, rotateConfig)
 }
 
 func InitDaemon(verbose, console bool, dir string, rotateConfig LogRotateConfig) error {
 	if console {
-		return createConsoleLogger(verbose)
+		return createConsoleLogger(verbose, "")
```

Comment:
InitDaemon, InitDfget, InitDfcache are not affected for now, by passing emty string here.